### DISCUSSION TITLE
Should error be caught from next/image in server render ?

### DISCUSSION THREAD

**Discussion by cai-zhuo** [vercel/next.js#59900] [App Router]
<img width="1643" alt="image" src="https://github.com/vercel/next.js/assets/44321355/b58d126c-484c-4897-8267-cc6168a490b4">

I have met this error in development environment, I specify a url in next/image such as `<Image src="xx.com/xx" />`, the url will be transform to "/next/_img?url=xx.com/xx", and the remote url will be fetched in server.

But when fetching is timeout, <Image /> will throw above error, and fallback to error.tsx.
Should I wrap a ErrorBoundary File for <Image / ? Or there is some configuration to disable remote fetching in server ?

**Comment by matthewraaff** [comment]
Have a look at custom image [loaderfiles](https://nextjs.org/docs/pages/api-reference/components/image#loaderfile). Here's a modified loaderfile/config based on the example.

```js
'use client'
 
export default function myImageLoader({ src, width, quality }) {
  return `{src}`
}

// Then in your next.config.js
module.exports = {
  images: {
    loader: 'custom',
    loaderFile: './my/image/loader.js',
  },
}
```
